### PR TITLE
Update D:\Git\winget-pkgs\manifests\j\jqlang\jq\1.6

### PR DESCRIPTION
**Pull request by Trenly**
###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/114410&drop=dogfoodAlpha

### CODE REVIEW

**Review by ImJoakim — APPROVED** [review]

File: manifests/j/jqlang/jq/1.6/jqlang.jq.installer.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
-# Created using wingetcreate 1.1.2.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+# Created with YamlCreate.ps1 v2.2.9 $debug=AUSU.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
 
 PackageIdentifier: jqlang.jq
-PackageVersion: 1.6
+PackageVersion: "1.6"
+InstallerType: portable
+Commands:
```
Comment (RockyMM) [inline manifests/j/jqlang/jq/1.6/jqlang.jq.installer.yaml:7]: This directive is mislocated. It should got under "Installers" directive according to the schema: https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.4.0/installer.md

File: manifests/j/jqlang/jq/1.6/jqlang.jq.installer.yaml [hunk 2]
```diff
@@ -1,14 +1,14 @@
-# Created using wingetcreate 1.1.2.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+# Created with YamlCreate.ps1 v2.2.9 $debug=AUSU.CRLF.5-1-19041-3031.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
 
 PackageIdentifier: jqlang.jq
-PackageVersion: 1.6
+PackageVersion: "1.6"
+InstallerType: portable
```
Comment (RockyMM) [inline manifests/j/jqlang/jq/1.6/jqlang.jq.installer.yaml:6]: This directive is mislocated. It should go under `AppsAndFeaturesEntries`

**Review by RockyMM — COMMENTED** [review]
I will propose a PR to fix this.

I am only looking at manifest spec, I am unsure how to test this...

File: manifests/j/jqlang/jq/1.6/jqlang.jq.installer.yaml (hunk 2 above)
Comment (Trenly) [reply manifests/j/jqlang/jq/1.6/jqlang.jq.installer.yaml:6]: No, in this case it is correct. Installer type can appear at several levels and has different meaning depending on its location.

Comment (RockyMM) [reply manifests/j/jqlang/jq/1.6/jqlang.jq.installer.yaml:6]: Ah, I see. I will fix it in my PR.

**Comment by RockyMM** [comment]
> I will propose a PR to fix this.
> 
> I am only looking at manifest spec, I am unsure how to test this...

Here is the PR #116426

**Pull request merged**